WTWSMS/common/religions/00_religions.txt

```diff
@@ -1408,26 +1415,58 @@ muslim = {
 		uses_decadence = yes
 		uses_jizya_tax = yes
 	}
-	wahhabi = {
+	wahhabi = {  #this is the extremist sect practiced by ISIL, Al-Qaeda, and Saudi Arabia to this day, as such, it is extremely violent, as it remains totally unchanged from its inception in the seventh and eighth centuries
```

Comment:
Wahhabism in the mod doesn't fit with that description at start, so I think we will have to revert those changes.
 Add a line note